src/VideoJS.as

```diff
@@ -66,9 +66,10 @@ package{
         }
 
         private function registerExternalMethods():void{
-
+            ExternalInterface.marshallExceptions = true;
```

Comment:
Chose to leave it in since marshallExceptions works in both directions, so this will expose any flash errors on the javascript side, which could be useful